app/test_app/functional/api/test_mobile_auth_api.py

```diff
@@ -43,7 +44,7 @@ def test_request_api_token_golden_path_success(
 
     set_pin_response = test_client.post('/api/v1/auth/reset_password/',
                                         data=json.dumps(
-                                            dict(phone=user_phone, new_password='1234', one_time_code=code)),
+                                            dict(phone=user_phone, new_pin='1234', one_time_code=code)),
```

Comment:
could we pls add one more step to the test here to verify that the new pin works?

Reply:
Yep!